src/common/wincmn.cpp

```diff
@@ -102,7 +103,14 @@ bool IsInCaptureStack(wxWindowBase* win);
 
 // We consider 96 DPI to be the standard value, this is correct at least for
 // MSW, but could conceivably need adjustment for the other platforms.
-static const int BASELINE_DPI = 96;
+// Since Window Vista the default DPI is defined in WinUser.h.
+static const int BASELINE_DPI =
+#if defined(__WXMSW__) && defined(USER_DEFAULT_SCREEN_DPI)
```

Comment:
Is this really worth the extra `#if` check? I don't expect the value of `USER_DEFAULT_SCREEN_DPI` to change for a very, very long time, if ever.